prep-javascript-functions/main.js

```diff
@@ -0,0 +1,62 @@
+function addTwoNumbers(x, y) {
+  return x + y;
+}
+
+var sum = addTwoNumbers(4, 4);
+console.log('sum', sum);
+
+function converHoursToMinutes(x, y) {
+  return x * y;
+}
```

Comment:
Your parameter name should be something that makes sense for the function. So here using `hours` would be better than `x`. This function should only have one parameter. Finally to calculate hours to minutes you would do `hours * 60`.